### ISSUE TITLE
Åland wind capacity data source missing

### ISSUE THREAD

**Issue by q--** [electricitymap/electricitymap-contrib#4183]
- Right now Åland wind production is 36.8 MW, 222.22% of the configured installed capacity of 16.6 MW. This needs to be updated with whatever the current installed capacity is. 
  - This was ***partially*** fixed in https://github.com/electricitymap/electricitymap-contrib/pull/4234 by adding a new wind farm to the current capacity, but without a complete data source we don't know if we're still missing some. Oil and interconnectors are now correct.
- `DATA_SOURCES.md` doesn't have a a capacity data source for Åland. I looked in the various Åland-related PRs/commits/issues and didn't notice any either. ~~The [production map](http://www.kraftnat.ax/text2.con?iPage=28&iLan=1) has wind power production as a percentage of (presumably) installed capacity though, so if rounding errors aren't important that could be used to derive the installed capacity.~~ As [VIKTORVAV99](https://github.com/VIKTORVAV99) points out, it's probably the share of total production+imports, not capacity; so, not useful for this purpose.
  - Oil and interconnectors got a data source in https://github.com/electricitymap/electricitymap-contrib/pull/4234, wind still needs one.

Don't have time to look any further right now, so opening this issue with what I know now.

**Edit:** possible wind data source: https://tuulivoimayhdistys.fi/en/wind-power-in-finland/map

**Comment by VIKTORVAV99** [comment]
> The [production map](http://www.kraftnat.ax/text2.con?iPage=28&iLan=1) has wind power production as a percentage of (presumably) installed capacity though, so if rounding errors aren't important that could be used to derive the installed capacity.

I don't think it is percentage of installed capacity but rather percentage of current consumption including imports.

If you take the total consumption (total förbrukning) and multiply it with the percentage you get (almost) the exact figure for wind.

I guess we could use the highest recorded wind production as capacity if we can't find a source.

**Comment by jvarho** [comment]
I don't have a source for the current total capacity, but the increase is from a recently commissioned 42 MW wind park, as described at https://kraftnat.ax/

I think it would be more or less correct to add that capacity to the current number.

**Comment by VIKTORVAV99** [comment]
I think the best option for now is to increase the current wind capacity by 42 megawatt as you suggest @jvarho.

But I think this issue should remain open until we can find a good capacity source for Åland wind (and other) production capacities.

**Comment by q--** [comment]
(Slightly off-topic) Interestingly, it looks like Kraftnat.ax have launched a new website in the weeks since I filed this issue. The old data source is still available (for now), but the new website has [generation data](https://kraftnat.ax/driftcentralen/) in textual form, which should be easier to parse than the old image-based source.

Also, the source specified by the parser needs to be updated, as kraftnat.aland.fi is gone and kraftnat.ax appears to be the new domain.

**Comment by VIKTORVAV99** [comment]
> (Slightly off-topic) Interestingly, it looks like Kraftnat.ax have launched a new website in the weeks since I filed this issue. The old data source is still available (for now), but the new website has [generation data](https://kraftnat.ax/driftcentralen/) in textual form, which should be easier to parse than the old image-based source.
> 
> Also, the source specified by the parser needs to be updated, as kraftnat.aland.fi is gone and kraftnat.ax appears to be the new domain.

I actually opened a separate issue for that at #4235 (it was missing some imports from Finland too which isa bigger immediate concern IMO.)
The parser source could be updated right away though, perhaps I should even include it in #4234?

**Comment by q--** [comment]
**TL;DR** oil 35MW, SE<->AX 80MW is correct, AX<->FI 10 MW + 100MW emergency backup link.

I found some data on fossil capacity. According to https://kraftnat.ax/om-kraftnat/ there are two diesel-fired power plants (of the gas turbine type), which serve as an emergency backup.

>In an emergency situation where connections to both Sweden and Finland are cut off, there are two gas turbines that can keep Åland's electricity network running in island operation. The gas turbines run on diesel oil which is stored for a few weeks of operation. However, the capacity is not sufficient for normal operation of the entire electricity grid in winter, without reducing the use of electricity.

According to https://kraftnat.ax/stamnatet/, these two have a capacity of 10 MW and 25 MW. So that makes for 35MW of oil capacity.

The latter website also mentions a 100MW link to Finland that is only meant to be used in an emergency, in addition to the 'regular' 10 MW Gustavskabeln that's used mostly during winter to top-up Swedish imports. The 80MW interconnector to Sweden is correct.

**Comment by q--** [comment]
Possible wind data source: https://tuulivoimayhdistys.fi/en/wind-power-in-finland/map

**Comment by VIKTORVAV99** [comment]
> Possible wind data source: https://tuulivoimayhdistys.fi/en/wind-power-in-finland/map

By adding up the locations that are "In Production" on Åland in that map and the newly commissioned Långnabba project (42 MW) I get the total figure to 63.8 MW, that sound like a reasonable figure to me.
